src/signals/incident-management/containers/Signaling/Signaling.tsx

```diff
@@ -49,7 +47,7 @@ const Signaling: FunctionComponent = () => {
   } = useGetReportReopenRequested()
 
   useEffect(() => {
-    getReportOpen({ end: endOpen })
+    getReportOpen({ end: new Date().toISOString() })
```

Comment:
The end parameter is optional in the API, so the addition is unnecessary. 

Looks like `getReportOpen` should be typed differently and/or set a default parameter to handle empty input.

Reply:
Good catch. Changed it.